Assets/Scripts/Object/Spawner.cs

```diff
@@ -16,7 +16,8 @@ public void FixedUpdate() {
         }
 
         private void SpawnRandomObject() {
-            AbstractObject rdObject = spawnableManager.objects[Random.Range(0, spawnableManager.objects.Length)];
+            AbstractObject[] objects = spawnableManager.GetObjects();
+            AbstractObject rdObject = objects[Random.Range(0, objects.Length)];
```

Comment:
![nits](https://img.shields.io/badge/review-nits-inactive.svg)
 
I prefer `randomObject` because it's clear to understand. 
(Is `rd` common abbreviation?)

Reply:
or if you have been used a lot of part in this code, it shouldn't be fixed because it takes time to replace

Reply:
okay changed that,
i think rd is a pretty common abbrev for random

Reply:
I see, I will make an announcement in discord so that everyone can recognize this abbreviation 😁